scripts/common_test.py

```diff
@@ -1389,3 +1390,24 @@ def mock_popen( # pylint: disable=unused-argument
             self.assertEqual(
                 common.start_subprocess_for_result(['cmd']),
                 (b'test\n', b''))
+
+    def test_workflow_permissions_set_to_read_all(self) -> None:
+        workflows_dir = '.github/workflows'
+        self.assertTrue(
+            os.path.isdir(workflows_dir),
+            f'{workflows_dir} directory not found.'
+        )
+
+        for filename in os.listdir(workflows_dir):
+            if filename.endswith('.yaml') or filename.endswith('.yml'):
```

Comment:
Note that you can pass a tuple to endswith(): .endswith(['.yaml', '.yml'])

Reply:
done